patroni/__init__.py

```diff
@@ -138,18 +139,41 @@ def shutdown(self):
 
 
 def patroni_main():
+    logdir = os.environ.get('PATRONI_FILE_LOG_DIR', None)
+    logsize = os.environ.get('PATRONI_FILE_LOG_SIZE', 25000000)
+    lognum = os.environ.get('PATRONI_FILE_LOG_NUM', 4)
     logformat = os.environ.get('PATRONI_LOGFORMAT', '%(asctime)s %(levelname)s: %(message)s')
     loglevel = os.environ.get('PATRONI_LOGLEVEL', 'INFO')
     requests_loglevel = os.environ.get('PATRONI_REQUESTS_LOGLEVEL', 'WARNING')
```

Comment:
`requests_loglevel` was introduced a very long time ago, when Patroni was mostly relying on python `requests` module and more important, `urllib3` was part of `requests` module those days.
Since then a lot of things changed, nowadays Patroni is mostly relying on `urllib3` directly, which was also separated from `requests`.
Basically it all means that `logging.getLogger('requests').setLevel(requests_loglevel)` is not doing anymore what it was intended to do and we should get rid of any explicit configuration for `requests` logging completely (`requests` is not logging anything anything).